db/csv/first_seed.csv

```diff
@@ -0,0 +1,42 @@
+title,description,Price,image_tag,Category

```

Comment:
this is the csv file! it has two categories and they each have products. 41 products total, all attributes should be included. At first glance, no nils spotted in seeded database.